extensions/2.0/Vendor/EXT_materials_specular_openpbr/README.md

```diff
@@ -0,0 +1,145 @@
+# EXT\_materials\_specular\_openpbr
+
+## Contributors
+
+- Mike Bond, Adobe, [@miibond](https://github.com/MiiBond)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+Must be defined as a sub-extension under the `KHR_materials_specular`
+
+## Exclusions
+
+- This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+- This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+Specular reflectance in glTF as defined by `KHR_materials_specular`  behaves a bit different than in OpenPBR. This extension defines an interpretation of the specular parameters that is compatible with OpenPBR's specular reflectance for both dielectrics and metals.
+
+This extension adds no additional parameters. Its presence merely changes how the parameters of `KHR_materials_specular` are used.
```

Comment:
PBR Material extensions are generally designed to be "as if always defined" with their default state not affecting the material appearance. To follow this convention, consider adding a boolean property (optional, false by default) that enables the new semantics.

Reply:
I've added a single boolean value to the extension to enable it.